### PR TITLE
POLIO-1084: Polio calendar export show wrong vaccines

### PR DESCRIPTION
**Pull request by hakifran**
Explain what problem this PR is resolving
- Polio calendar export show wrong vaccines
Related JIRA tickets : [POLIO-1084](https://bluesquare.atlassian.net/browse/POLIO-1084)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [x] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Filter on campaigns which has scopes

## How to test
- Go to the campaign calendar
- Export to xlsx

[POLIO-1084]: https://bluesquare.atlassian.net/browse/POLIO-1084?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by quang-le** [comment]
I added the missing FR keys and asked Rita to double check the PT

**Comment by hakifran** [comment]
@cypress

File: plugins/polio/api.py [hunk 1]
```diff
@@ -394,7 +394,7 @@ def get_round(self: "CampaignViewSet", round: Round, campaign: Campaign, country
         started_at = dt.datetime.strftime(round.started_at, "%Y-%m-%d") if round.started_at is not None else None
         ended_at = dt.datetime.strftime(round.ended_at, "%Y-%m-%d") if round.ended_at is not None else None
         obr_name = campaign.obr_name if campaign.obr_name is not None else ""
-        vacine = self.get_campain_vaccine(round, campaign)
+        vaccines = campaign.vaccines
```
Comment (quang-le) [inline plugins/polio/api.py]: this is incorrect when we have separate scopes per round.

You need to check if `separate_scope_per_round` is `True` for the campaign. If it's the case, you need to get the vaccines for the round. If not , you can use the campaign vaccines like you do here

We have this PR https://github.com/BLSQ/iaso/pull/683 that has been merged recently and allows you to use `Round.vaccine_names` to get the vaccines associated with round-level scopes

**Review by quang-le — CHANGES_REQUESTED** [review]
It doesn't work when you have separate scopes per round. See comment for possible solution.

I think you also need to update your tests to cover this case
https://github.com/BLSQ/iaso/assets/38907762/27fe1d79-adee-4d5c-8e88-6f7dae2589fb

**Review by quang-le — APPROVED** [review]
top!

**Review by madewulf — APPROVED** [review]

**Pull request merged**